onnxruntime/test/contrib_ops/rotary_embedding_op_test.cc

```diff
@@ -77,6 +77,7 @@ static void RunTest(
   if (tensor_type == TensorType::kFloat && !disable_cpu) {
     execution_providers.push_back(DefaultCpuExecutionProvider());
   }
+  execution_providers.push_back(DefaultWebGpuExecutionProvider());
```

Comment:
add `enable_webgpu` above (just like dml) so that this line does not break in non-webgpu build.